geoana/gravity.py

```diff
@@ -0,0 +1,269 @@
+"""
+======================================================
+Gravity (:mod:`geoana.gravity`)
+======================================================
+.. currentmodule:: geoana.gravity
+
+The ``geoana.gravity`` module contains simulation classes for solving
+basic gravitational problems.
+
+Simulation Classes
+==================
+.. autosummary::
+  :toctree: generated/
+
+  PointMass
+"""
+
+import numpy as np
+
+from scipy.constants import G
+
+
+class PointMass:
+    """Class for gravitational solutions for a point mass.
+
+    The ``PointMass`` class is used to analytically compute the gravitational
+    potentials, fields, and gradients for a point mass.
+
+        Parameters
+        ----------
+        mass : float
+            Mass of the point particle (kg). Default is m = 1 kg
+        location : array_like, optional
+            Location of the point mass in 3D space. Default is (0, 0, 0)
+        """
```

Comment:
Possible indentation error. Parameters should line up with the rest of the contents.